### PR TITLE
Translate installing/index.po

### PR DESCRIPTION
**Pull request by StevenHsuYL**
1. Complete installing/index.po
2. Fix a term "開關參數" in tutorial/modules.po

Thanks for review!

Reference: https://terms.naer.edu.tw/

### CODE REVIEW

File: installing/index.po [hunk 1]
```diff
@@ -285,40 +360,52 @@ msgid ""
...
+"::"
 
 #: ../../installing/index.rst:216
 msgid ""
 "There are also additional resources for `installing pip. <https://packaging."
 "python.org/tutorials/installing-packages/#install-pip-setuptools-and-"
 "wheel>`__"
 msgstr ""
+"關於\\ `安裝 pip <https://packaging.python.org/tutorials/installing-packages/"
+"#install-pip-setuptools-and-wheel>`__\\ ，這裡還有其他的資源。"
```
Comment (mattwang44) [inline installing/index.po]:
```suggestion
"這裡還有其他關於\\ `安裝 pip <https://packaging.python.org/tutorials/"
"installing-packages/#install-pip-setuptools-and-wheel>`__\\ 的資源。"
```

**Review by mattwang44 — COMMENTED** [review]
Everything looks great to me! Left one nitpicking comment.

**Comment by StevenHsuYL** [comment]
Thanks for the review!

**Review by mattwang44 — APPROVED** [review]
LGTM

**Pull request merged**